client/src/pages/EpisodeShowPage/index.tsx

```diff
@@ -582,12 +582,23 @@ export class WorkspaceView extends React.Component<any, any> {
     const doesRootWorkspaceScratchpadValueContainOneNode =
       _.get(rootWorkspaceScratchPad, "value[0].nodes.length") === 1;
 
-    const isRootWorkspaceScratchpadValueFirstTextNodeEmpty =
-      _.get(rootWorkspaceScratchPad, "value[0].nodes[0].leaves[0].text") === "";
+    const rootWorkspaceScratchpadFirstText = _.get(
+      rootWorkspaceScratchPad,
+      "value[0].nodes[0].leaves[0].text",
+    );
+
+    const isRootWorkspaceScratchpadFirstTextEmpty =
+      rootWorkspaceScratchpadFirstText === "";
+
+    const isRootWorkspaceScratchpadFirstTextDefaultValue =
+      typeof rootWorkspaceScratchpadFirstText === "string" &&
+      rootWorkspaceScratchpadFirstText.trim().toLowerCase() ===
+        "root-level scratchpad";
 
     const doesRootWorkspaceScratchpadContainRelevantContent = !(
       doesRootWorkspaceScratchpadValueContainOneNode &&
-      isRootWorkspaceScratchpadValueFirstTextNodeEmpty
+      (isRootWorkspaceScratchpadFirstTextEmpty ||
+        isRootWorkspaceScratchpadFirstTextDefaultValue)
     );
```

Comment:
Although nice and compact, this logic is hard to follow. Maybe something like this instead:
```javascript
doesRootWorkspaceScratchpadContainRelevantContent = !isRootWorkspaceScratchpadFirstTextEmpty && 
                                                    !isRootWorkspaceScratchpadFirstTextDefaultValue && 
                                                    !doesRootWorkspaceScratchpadValueContainOneNode
```

Could you also add a comment to explain the cases where `doesRootWorkspaceScratchpadValueContainOneNode` would be true? Without using Slate extensively, this is unclear.